leaderboard/service/remove_members.go

```diff
@@ -0,0 +1,14 @@
+package service
+
+import "context"
+
+const removeMembersServiceLabel = "remove members"
+
+// RemoveMembers reurn how many members have in a leaderboard
```

Comment:
I don't think this comment is right. RemoveMembers doesn't return the number of members in the leaderboard.

Reply:
Lol